astroquery/alma/tests/test_alma_remote.py

```diff
@@ -619,7 +519,7 @@ def test_staging_postfeb2020(alma):
 @pytest.mark.remote_data
 @pytest.mark.skip('Not working for now - Investigating')
 def test_staging_uptofeb2020(alma):
-    tbl = alma.stage_data('uid://A001/X121/X4ba')
```

Comment:
do we still need this test function and the previous, `test_staging_postfeb2020`? 2 year have passed since that API change, I suspect these can now be removed.